### PR TITLE
packery

### LINKED ISSUE #287: Draggable target is far from where the component hovers
### Warframe Hub Issue
---

**Summary (short):** Moving components feels jumpy.

---
**Description:** When clicking the header on a component, it jumps down instead of staying near the cursor.

Seems to be caused by `vue-responsive-grid-layout`

---
**If issue, Reproduction:** Drag a component

---
**Mockups, screenshots, evidence:**

---

(Check one)
- [x] Issue
- [ ] Suggestion

### LINKED ISSUE #288: Expanding accordions makes them go behind other components
### Warframe Hub Issue
---

**Summary (short):** Expanding collapsibles makes them go behind components that are "in front" or have the same/higher z-index, of the collapsibles.

Potential fix:  

---
**If issue, Reproduction:**
Open a bounty collapsible: turn [this](https://github.com/WFCD/warframe-hub/blob/f848741238d632bc93708d81281e29dfd5dd36f9/src/views/Timer.vue#L162) into a common function and call it on collapse/expand to force "reflow".

---
**Mockups, screenshots, evidence:**
1.  

2. 

---

(Check one)
- [x] Issue
- [ ] Suggestion

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary (short):**
switch back to packery

---
**Description:**
switching back to packery
todo:
~~- [ ] draggabilly~~
~~- [ ] store layoutf~~
~~- [ ] toggle dragging off (yay for mobile!)~~
- [x] fix nightwave #331 

---
**Fixes issue (include link):**
resolves #288 
resolves #287 

---
**Mockups, screenshots, evidence:**
see deployment

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "vue-analytics": "^5.17.0",
     "vue-native-notification": "^1.0.5",
     "vue-notification": "^1.3.16",
+    "vue-packery-plugin": "^1.4.8",
     "vue-raven": "^1.0.3",
     "vue-responsive-grid-layout": "^1.1.8",
```
Comment (EricSihaoLin) [inline package.json]: we can attempt to remove this now

File: src/store.js [hunk 2]
```diff
@@ -24,7 +23,7 @@ const state = {
   },
   platform: 'pc',
   theme: 'night',
-  grid: grid,
+  grid: {},
```
Comment (EricSihaoLin) [inline src/store.js]: do we still need this variable?

File: package.json (hunk 1 above)
Comment (TobiTenno) [reply package.json]: yeah, just forgot to uninstall

File: src/store.js (hunk 2 above)
Comment (TobiTenno) [reply src/store.js]: i was gonna store the new grid in it for ease.

Comment (TobiTenno) [reply src/store.js]: like positions, and whatnot

**Comment by TobiTenno** [comment]
I split the remaining todos to their own issue #333

File: src/store.js (hunk 2 above)
Comment (TobiTenno) [reply src/store.js]: I can delete for now since we won't use it for a while

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**